resources/views/admin/orders/list.blade.php

```diff
@@ -108,37 +100,9 @@
 				fill: false,
 				
 				data: [
-					{{ $monthly_invoices['1'] }},
-					{{ $monthly_invoices['2'] }},
-					{{ $monthly_invoices['3'] }},
-					{{ $monthly_invoices['4'] }},
-					{{ $monthly_invoices['5'] }},
-					{{ $monthly_invoices['6'] }},
-					{{ $monthly_invoices['7'] }},
-					{{ $monthly_invoices['8'] }},
-					{{ $monthly_invoices['9'] }},
-					{{ $monthly_invoices['10'] }},
-					{{ $monthly_invoices['11'] }},
-					{{ $monthly_invoices['12'] }},
-					{{ $monthly_invoices['13'] }},
-					{{ $monthly_invoices['14'] }},
-					{{ $monthly_invoices['15'] }},
-					{{ $monthly_invoices['16'] }},
-					{{ $monthly_invoices['17'] }},
-					{{ $monthly_invoices['18'] }},
-					{{ $monthly_invoices['19'] }},
-					{{ $monthly_invoices['20'] }},
-					{{ $monthly_invoices['21'] }},
-					{{ $monthly_invoices['22'] }},
-					{{ $monthly_invoices['23'] }},
-					{{ $monthly_invoices['24'] }},
-					{{ $monthly_invoices['25'] }},
-					{{ $monthly_invoices['26'] }},
-					{{ $monthly_invoices['27'] }},
-					{{ $monthly_invoices['28'] }},
-					{{ $monthly_invoices['29'] }},
-					{{ $monthly_invoices['30'] }},
-					{{ $monthly_invoices['31'] }}
+					@foreach($monthly_invoices as $invoice)
```

Comment:
Same as above, we don't even have to care about how many days there are in the month at this point.